Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -84,6 +84,20 @@ AVFoundationEnabled:
       "PLATFORM(WATCHOS)": false
       default: true
 
+AbortSignalAnyOperationEnabled:
+  type: bool
+  category: dom
+  status: preview
+  humanReadableName: "AbortSignal.any() operation"
+  humanReadableDescription: "AbortSignal.any() operation"
+  defaultValue:
+    WebKitLegacy:
+      default: false
+    WebKit:
+      default: false
+    WebCore:
+      default: false
```

Comment:
How does it work that the tests pass? Are features always enabled for tests?

Reply:
Yes, features get enabled for tests.

Reply:
Since I marked it as “preview”, I believe it would get enabled for Safari Technology Preview as well.

Reply:
This is correct. `testable` is the first stage where we activate features while running layout tests. `preview` takes it a step further, and activates for Safari Technology Preview.